log_file.log

```diff
@@ -0,0 +1,4 @@
+2019-04-11 16:33:52 ERROR ConnectorUtil:34 - Errorjava.io.FileNotFoundException: \database.properties (Не удается найти указанный файл)
```

Comment:
Please explain how does it work if all the time we can see such messages in your log file.

Reply:
Logger send this message when i tried to use url from database.properties

Reply:
Thia version isn't ready, so this time program doesn't work, later I'll try to fix this.